### ISSUE TITLE
Missing peaceful/skyblock recipes.

### ISSUE THREAD

**Issue by xyrc1** [MindBrain27/Sky-Greg#13]
Crying Obsidian is required to upgrade to Dank 6, and it is only obtainable by bartering with Piglins, which is not possible in Peaceful mode. Some kind of Chemical Reactor or Mixer recipe would make it possible for Peaceful players to get Dank 6s.

**Comment by MindBrain27** [comment]
Good call, added something in the Chemical Reactor in MV age. Any other missing vanilla/skyblock recipes you find but here as well, I'm going to pin this and change the title as a running issue.

**Comment by MindBrain27** [comment]
I have also added Chemical Reactor recipes for Leather, Moss Block, Lily Pad, Cobwebs, I'm sure there are a ton missing, most plant life will be added to Market block.

**Comment by TheMilso** [comment]
Leather you can get from a cow farm.  you can do golden egg to get A LOT of animals or a large grass field.

Moss Blocks you can get my using the grass seed to mud and mud you get from using dirt on the water bucket block you start with. (FYI don't let your kids sneak near the moss as they spread and create a giant forest out of your grass field)

**Comment by MindBrain27** [comment]
Leather recipe is more for a peaceful option, but I completely forgot Moss was already handled lol. There are some more being added like Lily Pads and Cobwebs and random world gen things next update.

Also I am going to blacklist Moss Blocks from the twerking mod because that sounds hilarious and also annoying at the same time lol.

**Comment by xyrc1** [comment]
As far as I'm aware Nether Stars are also unobtainable other than setting up a Wither farm in non-peaceful mode, which are needed some EV+ components and Dank 7.

**Comment by MindBrain27** [comment]
Got it, I added one this morning for that with an EV Chemical Reactor for crafting the Nether Star Dust, there's already an Implosion recipe for Nether Star Dusts to Nether Stars too. Should be an update tomorrow.

**Comment by TheMilso** [comment]
I cant see a way to get a smithing template.
Only one i see is when you have one.
![image](https://github.com/MindBrain27/Sky-Greg/assets/699986/8b22a577-4be9-4de8-b226-0b41f973e7b0)

**Comment by MindBrain27** [comment]
You can currently find them in the chests in Nether Fortresses but I don't really like that method at all, I'll add something in for it.

**Comment by MindBrain27** [comment]
Added a rather easy Assembler recipe.

**Comment by gaminggirl** [comment]
In Multiblocks there is a quest for a Hydrovolcanism Generator, but there is no recipe to actually make it.

**Comment by MindBrain27** [comment]
it will either be in next update or one after, still want to tune it a bit
